From 34af4105673a5f06063a953633c4b91a185863c0 Mon Sep 17 00:00:00 2001
From: Vyacheslav Bocharov <adeep@lexina.in>
Date: Thu, 29 Sep 2022 20:21:45 +0300
Subject: [PATCH 54/54] fix definition struct wifi_station
 *wifi_mac_fake_adhos_sta(struct wlan_net_vif *wnet_vif, const unsigned char
 macaddr[WIFINET_ADDR_LEN]);

---
 drivers/net/wireless/w1/vmac/wifi_mac_sta.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/w1/vmac/wifi_mac_sta.h b/drivers/net/wireless/w1/vmac/wifi_mac_sta.h
index ff7dbd089d24..6e26ba08c2aa 100644
--- a/drivers/net/wireless/w1/vmac/wifi_mac_sta.h
+++ b/drivers/net/wireless/w1/vmac/wifi_mac_sta.h
@@ -413,7 +413,7 @@ typedef void wifi_mac_IterFunc(void *, struct wifi_station *);
 void wifi_mac_func_to_task(struct wifi_station_tbl *, wifi_mac_IterFunc *, void *,unsigned char btask);
 void wifi_mac_disassoc_all_sta(struct wlan_net_vif*, wifi_mac_IterFunc*, void*);
 void wifi_mac_dump_sta(struct wifi_station_tbl *, struct wifi_station *);
-struct wifi_station *wifi_mac_fake_adhos_sta(struct wlan_net_vif *wnet_vif, const unsigned char macaddr[]);
+struct wifi_station *wifi_mac_fake_adhos_sta(struct wlan_net_vif *wnet_vif, const unsigned char macaddr[WIFINET_ADDR_LEN]);
 struct wifi_mac_scan_param;
 struct wifi_station *wifi_mac_add_neighbor(struct wlan_net_vif *, const struct wifi_frame *, const struct wifi_mac_scan_param *);
 
-- 
2.30.2

